src/js/plugins/menu_toggle/plugin.menu_toggle.js

```diff
@@ -188,6 +192,35 @@
       toggleRouter(br, e, atBookCenter);
     }
 
+    var initialX;
+    var initialY;
+    var holdOffOnToggle = false;
+    /**
+     * attaches mouseup & mousedown event handlers to assess if user is dragging
+     * sets `initialX`, `initialY`, and `holdOffOnToggle`
+     *
+     * `holdOffOnToggle` is used in fn `toggleRouter`
+     * to determine if menu toggle should happen
+     */
+    var registerDragHandlers = function registerDragHandlers() {
+      var background = document.querySelector('.BookReader') || {};
```

Comment:
QUESTION: If there's no `.BookReader` element and `background` becomes `{}`, what happens with `addEventListener()`? Anything bad?

Reply:
good question. main thing about the plugin is nothing instantiates unless bookreader is actually onscreen.
`|| {}` is a devving artifact. removed.
+ added early escapes